package.json

```
@@ -12,10 +12,13 @@
 		"build": "npm run build-clean && npm run build-compile",
 		"build-clean": "rm -rf dist",
 		"build-compile": "tsc",
+		"clean": "rm -rf coverage* && rm -rf dist && rm -rf node_modules && rm -f *.tgz",
```

Perhaps run the build-clean command rather than duplicating it's contents (although admittedly it's only contains 1 thing)
```suggestion
		"clean": "rm -rf coverage* && rm -rf node_modules && rm -f *.tgz && npm run build-clean",
```

---

Leaving this as it is as there's no functional difference.